bin/run_snd.py

```diff
@@ -24,11 +24,11 @@
 ##########
 # Module #
 ##########
```

Comment:
Minor comment: I'm currently trying to convince everyone to stop including these import headers. They serve no purpose, it should be implied that we organize imports into groups by kind as described in pep8 https://www.python.org/dev/peps/pep-0008/#imports with just a single blank line between them.

Reply:
Ah okay, I only started using them because I saw them in scripts you guys were writing. I'll start cleaning them up.

Reply:
It was me!